storage/remote/client.go

```diff
@@ -55,6 +77,11 @@ type ClientConfig struct {
 	HTTPClientConfig config_util.HTTPClientConfig
 }
 
+// Initalize Counter
```

Comment:
we don't need a comment here until some point in the future when we're maybe doing something more complex in `init` :)